_overviews/scala3-book/scala-features.md

```diff
@@ -30,8 +30,8 @@ Looking at Scala from the proverbial “30,000 foot view,” you can make the fo
 - It has a concise, readable syntax
 - It’s statically-typed (but feels dynamic)
 - It has an expressive type system
-- It’s a pure functional programming (FP) language
-- It’s a pure object-oriented programming (OOP) language
+- It’s a functional programming (FP) language
+- It’s an object-oriented programming (OOP) language
```

Comment:
I don’t think “pure” has a clear meaning for OOP, and Scala is clearly not a purely functional language.

Reply:
I went through some debate here about Line 33, but the short story is that this is fine.